python/sglang/srt/managers/controller/tp_worker.py

```diff
@@ -273,11 +273,12 @@ def handle_generate_request(
         req = Req(recv_req.rid, recv_req.input_text, recv_req.input_ids)
         req.pixel_values = recv_req.pixel_values
         if req.pixel_values is not None:
+            img_hash = sum(recv_req.image_hash) if type(recv_req.image_hash) is list else recv_req.image_hash
```

Comment:
Do one of the following
1. 
```suggestion
            img_hash = hash(recv_req.image_hash) if type(recv_req.image_hash) is list else recv_req.image_hash
```
2. pre-process the hash in the tokenizer_manager.py